python/amici/sbml_import.py

```diff
@@ -1438,8 +1344,8 @@ def _replace_in_all_expressions(self,
         """
         # Avoid replacing variables with rates
         if include_rate_rule_targets \
-                or old not in set((*self.compartment_rate_rules.keys(),
-                           *self.species_rate_rules.keys())):
+                or old not in {*self.compartment_rate_rules.keys(),
+                               *self.species_rate_rules.keys()}:
```

Comment:
Can also be without `.keys()` i.e. `{*self.compartment_rate_rules, *self.species_rate_rules}`

Reply:
👍